client/tests/doubles/client_api_stub.py

```diff
@@ -0,0 +1,23 @@
+from fenix import Event
+
+class ClientAPIStub:
```

Comment:
Docstrings?

Reply:
Do we need docstrings in the stubs? There isn't any business logic there so there isn't really any substance to document.